client/components/Tests/TestExercise.js

```diff
@@ -234,6 +234,7 @@ export const Editor = (props) => {
       })
       .then((res) => {
         setHasTestPassed(false);
+        console.log(res.data);

```

Comment:
I know this isn't being released to the public, but out of good habit, please delete or at least comment out console logs. I've been told to remove them out of security risks.

Reply:
Plus the data is in the networks tab if there is a need to debug it.